test/spec/services/situationService.js

```diff
@@ -232,4 +232,77 @@ describe('Service: situationService', function () {
             expect(results[0]).toBeFalsy();
         });
     });
+    describe('function setConjoint()', function() {
+        it('should add the conjoint at the end of the individus array', function() {
+            // given
+            var situation = {
+                individus:
+                    [
+                        { role: 'demandeur' },
+                        { role: 'enfant' }
+                    ]
+            };
+            var conjoint = { role : 'conjoint' };
+
+            // when
+            service.setConjoint(situation, conjoint);
+            // then
+            expect(situation.individus[2].role).toBe('conjoint');
```

Comment:
Meh. Really, this is testing and rigidifying behaviour that's broken anyway. Is there any domain or architectural reason for storing the Conjoint at the end of all Individus? If there isn't any, it's just a matter of fragile, improper modelling. Testing this is just testing a private implementation detail.

Put otherwise: eating that the Conjoint is replaced if one exists, or that it is properly available once set (if possible, through getters) adds value. Testing that the Conjoint is the last element in an array doesn't add any, and actually decreases our ability to make architectural improvements later on.